Gemfile.lock

```diff
@@ -368,6 +368,3 @@ DEPENDENCIES
   tzinfo-data
   uglifier (>= 1.3.0)
   w3c_validators
-
-BUNDLED WITH
-   1.10.6
```

Comment:
Oops! I think it got over written when I installed all the gem dependencies on my local system. Thanks for pointing it out. :+1: